openstack-scripts/prepare-tests.sh

```diff
@@ -63,6 +63,50 @@ function populate-kube-env () {
     sed -i "s/KUBE_TEST_REPO_LIST=/&\/home\/ubuntu\/run-e2e\/repo-list.yaml/" run-e2e/kube-env
 }
 
+
+function started() {
+    local TIME=`date +%s`
+    local GIT_VERSION=`git --version`
+    #local PULL_REFS=1
+
+    # making sure the tmp file is empty
+    echo "" > tmp.json
```

Comment:
Why? I would assume in case it already exists, but why would it if each job is unique? Also, where are we creating this tmp.json ?

Reply:
the echo command creates it automatically, but it would be more descriptive if I added a "touch tmp.json" before